scripts/build-element.js

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Built element scripts are missing "Banner"/preamble  in the beginning of the file